holidays/countries/faroe_islands.py

```
@@ -0,0 +1,103 @@
+        # Boxing Day - December 26
+        self._add_christmas_day_two(tr("Annar jóladagur"))
+
+        # New Year's Eve - December 31
+        self._add_new_years_eve(tr("Nýggjársaftan"))
+
+    def _populate_optional_holidays(self):
+        # Add optional holidays here if needed
+        pass
+
+
+class FO(FaroeIslands):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)